led_strip/include/led_strip.h

```diff
@@ -18,6 +18,20 @@
 extern "C" {
 #endif
 
+/**
+ * @brief Help macro to adjust pixel RGB color order
+ *        The default order of the three-color LED strips is GRB. If you have a different order, you can use the macro to set `config_pixel_order` in led_strip_config_t.
+ * @note The order starts from 0. And the user needs to make sure that all the numbers appear exactly once and are all less than the number of colors per pixel.
+ */
+#define LED_STRIP_ADJUST_RGB_ORDER(R, G, B) (R << 0 | G << 2 | B << 4)
```

Comment:
please add some comment, what's the meaning of the value R G B, should we count the order from MSB or LSB?